configs.js

```diff
@@ -0,0 +1,42 @@
+let alert = document.getElementById('alert');
+
+function showAlert(){
```

Comment:
Could you give me a little context on why this alert is needed and also why it needs a timeout? Sorry, my brain isn't at its best at the moment 😆